copilot/fastapi/app/config.py

```
@@ -0,0 +1,91 @@
+    This function attempts to load the in-cluster Kubernetes configuration.
+    If that fails, it falls back to loading the local kube config file.
+    """
+    try:
+        # Try to load in-cluster config (when running inside a pod)
+        config.load_incluster_config()
+    except config.ConfigException:
+        # Fall back to local kubeconfig for development
+        try:
+            config.load_kube_config()
+        except Exception as e:
+            raise HTTPException(status_code=500, detail=f"Failed to load Kubernetes config: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)